packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.tsx

```diff
@@ -614,13 +637,17 @@ class MarkdownSynapse extends React.Component<
     )
     if (renderInline) {
       return (
-        <span className="markdown markdown-inline" ref={this.markupRef}>
+        <span
+          aria-label="markdown"
```

Comment:
Added an `aria-label` to make this field easier to identify in tests

Reply:
I'm 50/50 on if this is a useful label for screen readers. Do you think this is a better `aria-label` or `data-testid`?